app/Config/Schema/schema.xml

```diff
@@ -1250,6 +1252,39 @@
     </index>
   </table>
   
+  <table name="co_provisioning_counts">
```

Comment:
I'm guessing this gets its own table because there's no existing equivalent of `co_jobs`? It'd be nice if both mechanisms used the same pattern (eg: `co_job_counts` to match `co_provisioning_counts`) but maybe we should wait until PE to fix that since the provisioning implementation is changing quite a bit anyway. We could maybe leverage the new `provisioning_history_records` table somehow. Although we do currently have `co_provisioning_exports`, maybe that could be used somehow?